.github/workflows/deploy.yml

```diff
@@ -317,10 +317,10 @@ jobs:
           retention-days: 30
 
   upload-reports:
-    name: Uppload Reports
+    name: Upload Reports
     needs:
       - test
-    if: always()
+    if: ${{ always() && github.ref_name != 'production' }}
```

Comment:
Do we even need the `always()` anymore?

Reply:
🤔 good question. I believe the purpose of the `always()` is to run this step even if previous steps fail. otherwise default behavior is to abandon all steps after a failure

Reply:
Makes sense! Thanks y'all